test/trivial/deitz/coerce-assign/plusassign_error.good

```diff
@@ -1 +1,8 @@
-plusassign_error.chpl:3: error: Cannot assign to int(32) from int(64)
+plusassign_error.chpl:3: error: unresolved call '+=(int(32), int(64))'
```

Comment:
Huh, this is a weird looking error. We went from "can't assign int32 to int64" to "+= on 8-bit ints didn't match". Not sure much can be done about this in the PR, though.